.coveragerc

```diff
@@ -0,0 +1,7 @@
+[run]
+source =
+	engine/
+    probtest.py
+	util/
```

Comment:
Should we add visualize/ as well here? I know it might be bad for the coverage.

Reply:
I added it. Makes sense to check this as well.